app/src/main/java/com/sun/basic_japanese/alphabet/adapter/AlphabetRecyclerAdapter.kt

```diff
@@ -0,0 +1,49 @@
+package com.sun.basic_japanese.alphabet.adapter
+
+import android.support.v7.widget.RecyclerView
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import com.sun.basic_japanese.R
+import com.sun.basic_japanese.constants.BasicJapaneseConstants.HIRAGANA
+import com.sun.basic_japanese.data.model.Alphabet
+
+class AlphabetRecyclerAdapter(
+    private val alphabetItems: MutableList<Alphabet?>,
```

Comment:
Ưu tiên non-null thì sẽ dễ xử lý hơn. Kiểm tra null ở nơi implement.
```suggestion
    private val alphabetItems: MutableList<Alphabet>,
```

Reply:
Em để null để thêm được các item rỗng với mục đích trình bày view theo cấu trúc bảng chữ cái nhật